lib/ansible/modules/apt.py

```diff
@@ -1321,7 +1321,7 @@ def main():
             module.fail_json(msg="Could not fetch updated apt files: %s" % fetchFailedException)
 
         # got here w/o exception and/or exit???
-        module.exit_json({'failed': True, 'msg': 'Unexpected code path taken,we really should have exited before, this is a bug'})
+        module.exit_json(**{'failed': True, 'msg': 'Unexpected code path taken, we really should have exited before, this is a bug'})
```

Comment:
```suggestion
        module.exit_json(failed=True, msg='Unexpected code path taken, we really should have exited before, this is a bug')
```
Feels a bit clunky to splat a dict instead of using the keyword form.

Reply:
Actually,
```suggestion
        module.fail_json(msg='Unexpected code path taken, we really should have exited before, this is a bug')
```